blog/2025-06-03-deprecate-verify-domain-control.md

```
@@ -0,0 +1,66 @@
+---
+author: epenet
+authorURL: https://github.com/epenet
+title: "Deprecate verify_domain_control service helper"
+---
+
+### Summary of changes
+
+The `homeassistant.helpers.service.verify_domain_control` helper function is deprecated,
+and has been replaced with `verify_domain_entity_control`.
+
+Since release `2025.1` (via core PR https://github.com/home-assistant/core/pull/133062),
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)